client/my-sites/stats/stats-email-top-row/index.jsx

```diff
@@ -23,6 +24,16 @@ export default function StatsEmailTopRow( { siteId, postId, statType, className
 		isRequestingEmailStats( state, siteId, postId, PERIOD_ALL_TIME, statType )
 	);
 
+	/**
+	 * Only show some email stats if post published more than 5 minutes ago.
+	 * As part of this, we fix the date sometimes object having wrong time of day
+	 * and normalize to utc to account for time zones differences.
+	 */
```

Comment:
Using date._i here seems fragile. That contains the original date string that was pulled passed to moment.js when the moment object was created. 

Presumably, we can assume it will always be there, but I should account for the possibility it doesn't. 

If it doesn't, we cannot reliably count on this moment object to have the accurate time. In that case, we can simply set emailIsSending to false, so the interface will render the number like before this PR. Or we can try an alternative way to get the post publish date/time.

Reply:
Actually, given that the _i here is an internal moment property, I wonder if it's best not to use that. Docs suggest using methods provided by moment, but that's the problem - they always give the wrong time because of time isn't parsed correctly upstream.

I may spend some time to explore other options here.

Reply:
So the time of day is wrong because upstream we explicitly set the date object to the start of the day here:

https://github.com/Automattic/wp-calypso/blob/c5823694f28980afe44e1ffa8842addc2337004c/client/state/stats/emails/utils.js#L8

I'll find some other work around for this so we don't use this cloned, start of day version of the time.